ShippoExample/Example.cs

```diff
@@ -20,122 +20,159 @@
+                                                            "CA", "94103", "US", "4151234567", "ship@gmail.com");
+            Address addressTo = Address.createForPurchase("Mrs. Hippo", "965 Missions St.", "Ste 202", "SF",
+                                                          "CA", "94103", "US", "4151234568", "msship@gmail.com");
+            Parcel[] parcels = {Parcel.createForShipment(5, 5, 5, "in", 2, "oz")};
+            Shipment shipment = Shipment.createForBatch(addressFrom, addressTo, parcels);
+            BatchShipment batchShipment = BatchShipment.createForBatchShipments(defaultCarrierAccount, "usps_priority", shipment);
+
+            List<BatchShipment> batchShipments = new List<BatchShipment>();
+            batchShipments.Add(batchShipment);
+
+            Batch batch = resource.CreateBatch(defaultCarrierAccount, "usps_priority", ShippoEnums.LabelFiletypes.PDF_4x6,
+                                               "BATCH #170", batchShipments);
+            Console.WriteLine("Batch Status = " + batch.Status);
+            Console.WriteLine("Metadata = " + batch.Metadata);
         }
 
-        private static void RunTrackingExample (APIResource resource)
+        private static void RunTrackingExample(APIResource resource)
         {
-            Track track = resource.RetrieveTracking ("usps", TRACKING_NO);
-            Console.WriteLine ("Carrier = " + track.Carrier.ToUpper ());
-            Console.WriteLine ("Tracking number = " + track.TrackingNumber);
+            Track track = resource.RetrieveTracking("usps", TRACKING_NO);
+            Console.WriteLine("Carrier = " + track.Carrier.ToUpper());
+            Console.WriteLine("Tracking number = " + track.TrackingNumber);
         }
 
-        static void Main (string[] args)
+        private static void RunInternationalAddressValidationExample(APIResource resource)
+        {
+            Hashtable parameters = new Hashtable();
+            parameters.Add("name", "Shippo Hippo");
+            parameters.Add("company", "Shippo");
+            parameters.Add("street_no", null);
+            parameters.Add("street1", "40 Bay St");
+            parameters.Add("street2", null);
+            parameters.Add("city", "Toronto");
+            parameters.Add("state", "ON");
+            parameters.Add("zip", "M5J 2X2");
+            parameters.Add("country", "CA");
+            parameters.Add("phone", "+1 555 341 9393");
+            parameters.Add("email", "hippo@goshippo.com");
+            parameters.Add("metadata", "Customer ID 123456");
+            parameters.Add("validate", "true");
+            Address address = resource.CreateAddress(parameters);
+            Console.Out.WriteLine("Address IsValid: " + address.ValidationResults.IsValid);
+            if (address.ValidationResults.Messages != null) {
+	            foreach (ValidationMessage message in address.ValidationResults.Messages) {
+	                Console.Out.WriteLine("Address Message Code: " + message.Code);
```

Comment:
Just to confirm, does our API now support these as separated fields in a JSON response?

Also, would this example break when using an older API Version?